lib/App.js

```diff
@@ -91,6 +104,27 @@ App.prototype._finishInit = function() {
   }
   this.emit('load', page);
 };
+
+App.prototype._isAttached = function isInitialized() {
+  const { attached } = globalThis.APPS.get(this.name);
```

Comment:
For now, let's stick with ES5 syntax for consistency with the existing code.
- `const` -> `var`
- No object destructuring